README.md

```diff
@@ -209,6 +209,45 @@ Methods that are mapped using `map_motion` accept an `event` parameter which is
 
 See the code for full API for [Event](https://github.com/unabridged/motion/blob/master/lib/motion/event.rb) and [Element](https://github.com/unabridged/motion/blob/master/lib/motion/element.rb).
 
+## Callbacks
+
+Motion has callbacks which will let you pass data from a child component back up to the parent. `bind` is how Motion figures out what method to call. `on_click` is a method passed into the child component and can be named whatever you'd like. When the component's motion is called, the motion calls the method in the parent component.
+
+**parent_component.rb**
+```ruby
+  map_motion :do_something
```

Comment:
Callbacks can be mapped as a motion, but they don't need to be, so I think we should exclude this line.

Reply:
Good to know, thank you.